modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -27,6 +27,38 @@ This illustration summarizes how each asset type (in green) appears in Exchange:
 
 image::ex2-exchange-assets.png[Illustration - How Assets appear in Exchange]
 
+== Asset Limits
+
+Exchange prevents resource exhaustion attacks by limiting the number of asset versions that can be published.
+
+The limit is 500 assets for master organizations with trial accounts, and 100 000 (one hundred thousand) assets for other master organizations.
```

Comment:
Is there a reason to use 100 000 instead of 100,000?

Reply:
Yes, the BA devs pointed out that many of our Latin American and European customers use 100,000 to mean "one hundred," while the US uses 100.000 to mean "one hundred." Likewise they would use 100.000 to mean "one hundred thousand" while the US uses 100,000 to mean "one hundred thousand." Since 2003 most groups have lined up behind the use of spaces, including NIST, the International Bureau of Weights and Measures, the AMA, and IUPAC. I learned all of this after the BA devs brought it up on this pull request. Full info: https://en.wikipedia.org/wiki/Decimal_separator